### PR TITLE
Doc Updates

### PR DESCRIPTION
**Pull request by dragazo**
This PR changes a bunch of files, but they're almost exclusively docstring changes.

I went through all the services and updated their docs to make them more informative and look better when viewed in the compiled html version. This includes updating/adding descriptions for RPCs, params, and return values, as well as adding types (only return types, I think, were missing). I also added more cross-referencing between rpcs (in the same service) where appropriate.

I cleaned up a bunch of the custom rst we have from the migrated wiki docs, including changing all (named) links to anonymous links, and using `*...*` instead of `` `...` `` for italics (latter is apparently platform-specific).

I noticed `MovieDB` had a *ton* of RPCs, so I split them into two categories: `Movies` and `People`. We might want to refine this at some point, but it's at least not a breaking change since it only affects the drop down display.

I fixed some rpc param types which were using the wrong casing version (e.g., `string -> String`).

I noticed docs were being generated for a service that was marked as deprecated, so I added the `tags` field to jsdoc metadata and added another (default) filter to ignore deprecated services. Also, I fixed a bug in the html doc gen where it was creating default rpc category rst files wrong (it had `>>>SERV<<<` instead of `>>>RPCS<<<`).

Also I fixed a bug that showed up in the `Geolocation` service where `jsdoc-extractor` wasn't extracting metadata for rpcs that were added with the indexing syntax: `ServiceName['rpcName'] = function ...`.

### CODE REVIEW

File: src/server/services/procedures/human-mortality-database/human-mortality-database.js [hunk 1]
```diff
@@ -173,23 +173,20 @@ async function getData() {
...
+ * Get all the mortality data. This is potentially a lot of data.
+ * **Only use this if you truly need access to all data.**
+ * 
+ * This is returned as structured data organized by country, then by date (mm/dd/yyyy), then by gender, then by category.
  *
- * @returns {Array}
+ * @returns {Array} all available data
  */
-mortality.getAllData = getData;
+mortality.getAllData = async () => await getData();
```
Comment (brollb) [inline src/server/services/procedures/human-mortality-database/human-mortality-database.js:183]: Why was this change necessary? It isn't a problem but I am curious...

**Review by brollb — APPROVED** [review]
Looks good. How do the updated docs look from the help menu in the NetsBlox UI? If it looks clunky, we could check out updating the help button to just open the documentation...

Check out the codeclimate issue (it looks like there is a minor indentation issue) and then it should be good to go as soon as the tests pass!

File: src/server/services/procedures/human-mortality-database/human-mortality-database.js (hunk 1 above)
Comment (dragazo) [reply src/server/services/procedures/human-mortality-database/human-mortality-database.js:183]: Oh, that was an issue where apparently the jsdoc extractor wasn't matching that syntax. I fixed it before the other jsdoc syntax thing and forgot about it. I can go back and just add another regex for alias rpcs like this.

**Comment by dragazo** [comment]
I added some code in the original PR for html docs that cleans up the rst syntax in the help menu strings, but they're definitely a lot longer now. One example I looked at was the help page for `Smithsonian.search`, which is much longer now, but has good formatting.
